pkg/api/types.go

```diff
@@ -14,6 +15,49 @@ const (
 	StatusError        OpenSCAPStatus = "Error"
 )
 
+// ScanResult represents the compacted result of all scans performed on the image
+type ScanResult struct {
+	// ImageName is a full pull spec of the input image
+	ImageName string `json:"imageName"`
+	// ImageIUD is a SHA256 identifier of the scanned image
+	ImageID string `json:"imageID"`
```

Comment:
@mfojtik In ManageIQ we are comparing the Id / repo digest of the image [1]

[1]https://github.com/ManageIQ/manageiq-providers-kubernetes/blob/master/app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb#L112